docs/src/main/asciidoc/_developer-guide/How_do_I_migrate_my_application_from_AS7_to_WildFly.adoc

```diff
@@ -21,7 +21,7 @@ The list of {ProductName} new functionality is extensive, being the most relevan
 * Apache ActiveMQ Artemis - {ProductName}'s new JMS broker. Based on an code donation from HornetQ, this Apache subproject provides outstanding performance based on a proven non-blocking architecture.
 * IronJacamar 1.2 - The latest IronJacamar provides a stable and feature rich JCA & Datasources support.
 * JBossWS 5 - The fifth generation of JBossWS, a major leap forward, brings new features and performances improvements to {ProductName} Web Services
-* RESTEasy 3 - {ProductName} includes the latest generation of RESTEasy, which goes beyond the standard Jakarta EE REST APIs (JAX-RS 2.0), by also providing a number of useful extensions, such as JSON Web Encryption, Jackson, Yaml, JSON-P, and Jettison.
+* RESTEasy 3 - {ProductName} includes the latest generation of RESTEasy, which goes beyond the standard Jakarta EE REST APIs (JAX-RS 2.0), by also providing a number of useful extensions, such as JSON Web Encryption, Jackson, Yaml, Jakarta JSON Processing, and Jettison.
```

Comment:
Actually, RESTEasy 3.x is no longer the "latest generation of RESTEasy". We're currently releasing 3.x and 4.x versions.